CONTRIBUTORS.md

```diff
@@ -23,4 +23,4 @@ Please prefix all commit messages in your PR with `#{issue_number}` to help with
 
 ## Making a PR
 
-Make a PR to commit your code to the master branch. At this point, at least one reviewer needs to sign off before the code can be merged. They may also have comments or requests for changes. Please work with the reviewer until they are satisfied. Once the approval is given, merge your branch via the `Squash and Merge` merge method.
\ No newline at end of file
+Make a PR to commit your code to the master branch. At this point, at least one reviewer needs to sign off before the code can be merged. They may also have comments or requests for changes. Please work with the reviewer until they are satisfied. Once the approval is given, merge your branch via the `Squash and Merge` merge method.
```

Comment:
What was changed here? I don't see it.

Reply:
To be honest I actually don't know either. It may have something to do with editing files through Github's website